app/controllers/api/v1/short_form_controller.rb

```diff
@@ -6,159 +6,35 @@ class Api::V1::ShortFormController < ApiController
 
   def submit
     logger.debug "application_api_params: #{application_api_params}"
-    short_form_validator = ShortFormValidator.new(application_api_params)
-
-    if short_form_validator.valid?
-      application = custom_api_application_service.submit(application_api_params)
-      logger.debug "application submit response: #{application}"
-      # if submitting a supplemental application we will re-fetch updated data from salesforce
-      if params[:supplemental]
-        application = soql_application_service.application(application[:id] || application_api_params[:id], { include_lease: false })
-        logger.debug "updated application: #{application}"
-      end
-
-      render json: { application: application }
-    else
-      render status: 422, json: { errors: short_form_validator.errors.full_messages || 'Unknown Error' }
+    # short_form_validator = ShortFormValidator.new(application_api_params)
+
+
+    # if short_form_validator.valid?
+    # FIXME: validatate / permit on the params
+    custom_api_application = Force::Application.from_domain(application_api_params).to_custom_api
+    puts 'Custom api application', custom_api_application.to_json
+    application = custom_api_application_service.submit(custom_api_application)
+    logger.debug "application submit response: #{application}"
+    puts "application submit response: #{application}"
+    # if submitting a supplemental application we will re-fetch updated data from salesforce
+    if params[:supplemental]
+      application = soql_application_service.application(application[:id] || application_api_params[:id], { include_lease: false })
+      logger.debug "updated application: #{application}"
     end
+
+    render json: { application: application }
+    # else
```

Comment:
this still needs to be cleaned up/fixed

Reply:
@software-project you're right, if we don't have the validator (and it wasn't really doing anything before) we can just remove this piece.